style.css

```diff
@@ -0,0 +1,107 @@
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 64px 24px;
+  gap: 58px;
+  position: absolute;
+  width: 375px;
+  height: 726px;
+  left: 0;
+  top: 92px;
+  background-image: url('background.png');
+  background-position-x: center;
+  background-position-y: center;
+  background-size: cover;
+  background-clip: inherit;
+}
+
+h2 {
+  color: rgb(27, 161, 45);
```

Comment:
- [optional] Please kindly change the color of this text according to the given template working according to design always give hiring managers confidence that we can deliver as expected. 👍